fast64_internal/oot/actor/properties.py

```diff
@@ -110,22 +163,189 @@ def draw_props(
 class OOTActorProperty(PropertyGroup):
     actorID: EnumProperty(name="Actor", items=ootData.actorData.ootEnumActorID, default="ACTOR_PLAYER")
     actorIDCustom: StringProperty(name="Actor ID", default="ACTOR_PLAYER")
+
     actorParam: StringProperty(name="Actor Parameter", default="0x0000")
     rotOverride: BoolProperty(name="Override Rotation", default=False)
-    rotOverrideX: StringProperty(name="Rot X", default="0")
-    rotOverrideY: StringProperty(name="Rot Y", default="0")
-    rotOverrideZ: StringProperty(name="Rot Z", default="0")
+    rotOverrideX: StringProperty(name="Rot X", default="0x0000")
+    rotOverrideY: StringProperty(name="Rot Y", default="0x0000")
+    rotOverrideZ: StringProperty(name="Rot Z", default="0x0000")
+
+    params: StringProperty(
+        name="Actor Parameter",
```

Comment:
could there be a comment somewhere about the difference between `actorParam` and `params`

Reply:
``actorParam`` is used for custom actors (actors with the id custom)

Reply:
both are a `StringProperty` without get/set, it doesn't seem there needs to be two?

Reply:
``params`` use get/set